source/Assets/Plugins/Didomi/Scripts/Purpose.cs

```diff
@@ -1,93 +1,31 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Runtime.Serialization;
+using Newtonsoft.Json;
 
 namespace IO.Didomi.SDK
 {
     [Serializable]
     public class Purpose
     {
-        private string id;
+       
+        [JsonProperty("id")]
+        public string Id { get; set; }
 
-        private string iabId;
+        [JsonProperty("name")]
+        public string Name { get; set; } = "";
 
-        private string name;
+        [JsonProperty("description")]
```

Comment:
The json property should be `descriptionText` as well

Reply:
thanks